config.yaml

```diff
@@ -33,9 +95,24 @@ weather:
             impulse_temp: 10 ## deg C
             impulse_duration: 60 ## seconds
             impulse_cycle: 600 ## seconds
+        thresholds:
+          sky-ambient:
+            cloudy: -25., -15.
```

Comment:
You need a 'clear' category e.g. `-.inf, -25`, otherwise `_get_status()` is going to mark clear conditions with the 'invalid' default status.

Reply:
Inlcuded the 'clear' category.

Reply:
You have for the AAT met data, but 'clear', 'calm', etc. are still missing for the local AAG section.